apps/annotations/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import pgettext_lazy
+from taggit.models import TagBase, TaggedItem
+
+from apps.teams.models import BaseTeamModel
+
+
+class Tag(TagBase, BaseTeamModel):
```

Comment:
Maybe getting ahead of myself but `tag_color` would be a nice to have (#scope-creep)